src/core/lv_group.h

```diff
@@ -22,7 +22,7 @@ extern "C" {
  *      DEFINES
  *********************/
 /*Predefined keys to control the focused object via lv_group_send(group, c)*/
-enum _lv_key_t {
+enum {
```

Comment:
removing the names and making enumerations anonymous is going to break the documentation.

Reply:
I agree that it should be `typedef enum{} lv_key_t;`

Reply:
I agree that it should be typedef enum{} lv_key_t;